### PR TITLE
Enable term editing

### PR DESCRIPTION
**Pull request by pdewouters**
it's not persisting the change actually

### CODE REVIEW

File: containers/App.js [hunk 1]
```diff
@@ -66,6 +66,12 @@ class App extends Component {
 		})
 	}
 
+	handleUpdateTerm() {
+		this.props.dispatch({
```
Comment (pdewouters) [inline containers/App.js]: @joehoyle this is where the term is saved ?

File: components/Terms/Edit.js [hunk 2]
```diff
@@ -12,21 +12,31 @@ export default class Edit extends Component {
 	static propTypes = {
 		term: PropTypes.Term.isRequired,
 	}
+
+	constructor(props) {
+		super(props)
+		this.state = {
```
Comment (joehoyle) [inline components/Terms/Edit.js]: @pdewouters why is this copying to the state? Let's just use the term directly in `this,props` in `render`

Comment (pdewouters) [inline components/Terms/Edit.js]: aren't props supposed to be readonly? 
I was following the example from http://facebook.github.io/react-native/releases/0.28/docs/textinput.html#textinput

screenshot 2016-07-01 11 26 19

**Comment by pdewouters** [comment]
@joehoyle I'll take a look this week still

File: components/Terms/Edit.js (hunk 2 above)
Comment (pdewouters) [inline components/Terms/Edit.js]: How would you edit the value without using state?

**Comment by pdewouters** [comment]
this has diverged too much, I'll open a new PR

**Pull request closed without merging**